apps/camera/js/lib/camera/camera.js

```diff
@@ -1251,8 +1307,13 @@ Camera.prototype.isMode = function(mode) {
  *
  * @private
  */
-Camera.prototype.startVideoTimer = function() {
-  this.set('videoStart', new Date().getTime());
+Camera.prototype.startVideoTimer = function(resume) {
+ if (resume) {
+    var delta = this.videoStopped - this.get('videoStart');
+    this.set('videoStart', (new Date().getTime()) - delta);
```

Comment:
Replace `new Date.getTime()` with `Date.now()`. It avoids creating a new instance of a `Date` object when all you're after is the epoch number.